src/screens/op-rewards/OpRewardsSheet.tsx

```diff
@@ -1,19 +1,38 @@
-import React from 'react';
+import React, { useCallback, useState } from 'react';
 import { SlackSheet } from '@/components/sheet';
 import { useDimensions } from '@/hooks';
 import { Box, Text } from '@/design-system';
 import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { useFocusEffect } from '@react-navigation/native';
+import { getMockOpData } from '@/screens/op-rewards/mocks/getMockOpData';
 
 export const OpRewardsSheet: React.FC = () => {
   const { height } = useDimensions();
   const { top } = useSafeAreaInsets();
+  const [rewardsData, setRewardsData] = useState<any>(null);
+  const [loading, setLoading] = useState(true);
+
+  useFocusEffect(
+    useCallback(() => {
```

Comment:
not super important since this is all mock but should prob use react query yeah?